packages/vscode-extension/extension.js

```diff
@@ -62,16 +82,26 @@ class ActivefileWatcher {
 				this.activefile = null;
 				this.activeProject = null;
 				this.language = null;
+				this.gitBranch = null;
```

Comment:
Can you explain if currentFile !== this.activefile, why would you set it activeFile to null after storing instead of setting it to currentFile.

Reply:
Yeah i got it now what I am doing wrong. I was setting it to null so that in next iteration, the activefile will be null so it will set the activefile now to the new currentfile which is open.